extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -149,33 +158,37 @@ public function execute() {
 				if ( $dbname && $folder ) {
 					$this->info( "Dumping images on remote host" );
 					try {
-						$source = $this->tarFiles( $dbname, $cityid );
-
-						if ( is_string( $source ) ) {
-							try {
-								DumpsOnDemand::putToAmazonS3( $source, !$hide,
-									MimeMagic::singleton()->guessMimeType( $source ) );
-							}
-							catch ( S3Exception $ex ) {
-								$this->error( "putToAmazonS3 command failed - Can't copy images to remote host. Please, fix that and rerun",
-									[
-										'exception' => $ex->getMessage(),
-										'dump_size_bytes' => filesize( $source ),
-									] );
+						foreach( $this->tarFiles( $dbname, $cityid ) as $source ) {
+							if ( is_string( $source ) ) {
+								try {
+									DumpsOnDemand::putToAmazonS3( $source, !$hide,
+										MimeMagic::singleton()->guessMimeType( $source ) );
+								}
+								catch ( S3Exception $ex ) {
+									$this->error( "putToAmazonS3 command failed - Can't copy images to remote host. Please, fix that and rerun",
+										[
+											'exception' => $ex->getMessage(),
+											'dump_size_bytes' => filesize( $source ),
+										] );
+									unlink( $source );
+
+									// SUS-6077 | move to a next wiki instead of failing the entire process
+									continue;
```

Comment:
after you added an extra loop - should this `continue` do to the next tarFile or to the next wiki (with possible cleanup of previous files)?

Reply:
Right. Fixed in 2dccc7d1ba0

Reply:
Why PHP does not have labeled loop statements? 🤔